fs/metrics/common/metrics.go

```diff
@@ -62,22 +62,61 @@ const (
 
 // Lists all metric labels.
 const (
-	// prometheus metrics
-	Mount                         = "mount"
-	RemoteRegistryGet             = "remote_registry_get"
-	NodeReaddir                   = "node_readdir"
-	InitMetadataStore             = "init_metadata_store"
-	ReadOnDemand                  = "read_on_demand"
-	MountLayerToLastOnDemandFetch = "mount_layer_to_last_on_demand_fetch"
+	/**
+	Metrics emitted via prometheus endpoint. All latency metrics are measured in milliseconds,
+	unless specified otherwise.
+	*/
+	// Mount measures the duration of Mount operation for resolved layer.
+	Mount = "mount"
+
+	// RemoteRegistryGet records the roundtrip latency for remote registry GET operation.
+	RemoteRegistryGet = "remote_registry_get"
+
+	// NodeReaddir measures how long readdir operation takes (in microseconds).
+	NodeReaddir = "node_readdir"
+
+	// InitMetadataStore measures how long it takes to initialize the file metadata store.
+	InitMetadataStore = "init_metadata_store"
+
+	// ReadOnDemand measures the latency of on-demand reads (in microseconds).
+	ReadOnDemand = "read_on_demand"
```

Comment:
"OnDemand" is a funny term. I think we will have three kinds of reads in the fullness of time. If we can think of names that would be particularly helpful to someone doing performance analysis, that would be great. Failing that, we should use the terms that our code uses. We don't use the term "on demand" anywhere except when referring to these metrics. 

I'll suggest:
* "Synchronous reads" or maybe "Blocking reads".
* "Prefetch reads" 
* "Background reads"
I don't feel that strongly about them.